geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientProxy.java

```diff
@@ -852,7 +860,6 @@ protected void terminateDispatching(boolean checkQueue) {
       return;
     }
 
-    boolean closedSocket = false;
```

Comment:
Barry added these changes to logout the subject at the end of this method. Later on we added "logging out subject" in the `close` method of the dispatcher, so these changes are not needed anymore. The test Barry added is still passing, so, we are not breaking anything. We revert these changes so that we manage the subject logout in one place.